app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala

```diff
@@ -58,7 +57,11 @@ object MoneyUtils {
     value: BigDecimal
   ): Either[FormError, BigDecimal] =
     if (value <= limitInclusive.inPounds()) Right(value)
-    else Left(FormError(key, errorMessageKey, List(limitInclusive.inPounds().toString())))
+    else {
+      Left(
+        FormError(key, errorMessageKey, List(limitInclusive.inPounds().toString()))
```

Comment:
the `List` in the error message might originate from here - what happens if you don't wrap the value in a list?

Reply:
Yes, it is already fixed. I was giving the wrong key, thanks